x-pack/legacy/plugins/code/public/monaco/monaco_helper.ts

```diff
@@ -48,6 +48,29 @@ export class MonacoHelper {
       if (chrome.getInjected('enableLangserversDeveloping', false) === true) {
         this.monaco.languages.registerDefinitionProvider('go', definitionProvider);
       }
+      const langsSupportedByCtags = [
```

Comment:
why don't you use `laugauge_servers.CTAG.languages` directly.

Reply:
When I import CTAGS from `language_servers`, the kibana will fail to optimize with the following error which is weird:

```
optmzr    log   [20:36:09.887] [fatal][optimize] Optimization failed in 3.93 secondsError: Optimizations failure.
   7440 modules

    ERROR in ./x-pack/legacy/plugins/code/server/lsp/ts_launcher.ts
    Module not found: Error: Can't resolve 'child_process' in '/Users/poytr1/workspace/kibana/x-pack/legacy/plugins/code/server/lsp'

    ERROR in ./x-pack/legacy/plugins/code/server/lsp/java_launcher.ts
    Module not found: Error: Can't resolve 'child_process' in '/Users/poytr1/workspace/kibana/x-pack/legacy/plugins/code/server/lsp'

    ERROR in ./x-pack/legacy/plugins/code/server/lsp/ctags_launcher.ts
    Module not found: Error: Can't resolve 'child_process' in '/Users/poytr1/workspace/kibana/x-pack/legacy/plugins/code/server/lsp'

    ERROR in ./x-pack/legacy/plugins/code/server/lsp/proxy.ts
```

Reply:
Oh, I understand.  Let do a little refactor here. Move this string array to somewhere in common folder.  For example in `common/language_server.ts`
```
export CTAG_SUPPORT_LANGS = '...'
```

then import them both in frontend and backend codes, thanks.

Reply:
Done.